.ci/build_images.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
```

Comment:
`set -euo pipefail` would avoid tracking the failures (the main script would exit with the first failing command exit code.

Reply:
`set -x` would display all the commands ran by the main script

Reply:
Not really in favor of `set -x` outside of debugging, but `set -e` is a must have and `set -uo pipefail` is nice to have.